packages/docs/content/components/accordion.mdx

```diff
@@ -1,26 +1,31 @@
 ---
 title: Accordion
 status: draft
+intro: An accordion is a list of headers that hide or reveal additional content when selected.
+core:
+  githubLink: design-system/src/components/Accordion/Accordion.tsx
```

Comment:
Curious - should we link to the tsx file or the directory of the component? Or does that even matter?

Reply:
I could go either way. I think for some, I linked directly to a file and others I linked to the directory. I chose files when the whole directory felt like it had an overwhelming amount of files. But I'm happy to update to whatever folks like

Reply:
Updating to link to the directory for v1.